releasenotes/notes/1.0/prepare-1.0-e13b9b8273055775.yaml

```
@@ -0,0 +1,69 @@
+---
+prelude: |
+  We are very please to release the Qiskit 1.0.0 release. This release is the
+  culmination of 7 years of development to mature Qiskit into a stable high
+  performance SDK to perform quantum computation, and is the start of new
+  era for the Qiskit project and community. The largest change with the release
+  is the changes to the stability policy, release cycles, and versioning.
+  Qiskit is now formally using `semantic versioning <https://semver.org/>`__
```

tried to find an alternative to avoid the "the largest change is the changes":

```suggestion
  era for the Qiskit project and community. The most significant changes with the 1.0.0 
 release relate to the stability policy, release cycles, and versioning.
  Qiskit is now formally using `semantic versioning <https://semver.org/>`__ ,
```